components/prompt_based_laion_retrieval/src/clip_client.py

```
@@ -7,6 +7,8 @@
 Copyright (c) 2021 Romain Beaumont
 """
 
+# mypy: ignore-errors
```

Ignore errors since this is a copied file.